cmake/projects/Boost/ci/matrix.json

```diff
@@ -126,6 +126,23 @@
 { "example": "Boost-filesystem-useBoostConfig", "toolchain": "mingw-cxx17",                                     "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
 { "example": "Boost-filesystem-useBoostConfig", "toolchain": "msys-cxx17",                                      "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
 
+{ "example": "Boost-json", "toolchain": "clang-cxx17",                                     "os": "ubuntu-16.04", "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "gcc-7-cxx17",                                     "os": "ubuntu-16.04", "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "gcc-8-cxx17-fpic",                                "os": "ubuntu-18.04", "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "gcc-9-cxx17-fpic",                                "os": "ubuntu-20.04", "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "android-ndk-r17-api-24-arm64-v8a-clang-libcxx14", "os": "ubuntu-16.04", "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "osx-10-13-make-cxx14",                            "os": "macos-10.15",  "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "osx-10-13-cxx14",                                 "os": "macos-10.15",  "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "osx-10-15-dep-10-10-cxx17",                       "os": "macos-10.15",  "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "ios-nocodesign-13-2-dep-9-3",                     "os": "macos-10.15",  "python": "3.8", "script": "build.sh"  },
+{ "example": "Boost-json", "toolchain": "ninja-vs-15-2017-win64-cxx17",                    "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
+{ "example": "Boost-json", "toolchain": "nmake-vs-15-2017-win64-cxx17",                    "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
+{ "example": "Boost-json", "toolchain": "vs-16-2019-win64-cxx17",                          "os": "windows-2019", "python": "3.8", "script": "build.cmd" },
+{ "example": "Boost-json", "toolchain": "vs-15-2017-win64-cxx17",                          "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
+{ "example": "Boost-json", "toolchain": "vs-14-2015-sdk-8-1",                              "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
+{ "example": "Boost-json", "toolchain": "mingw-cxx17",                                     "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
+{ "example": "Boost-json", "toolchain": "msys-cxx17",                                      "os": "windows-2016", "python": "3.8", "script": "build.cmd" },
+
```

Comment:
Unfortunately, this is to much for the github builds. I think, only 256 jobs are allowed for one "action". Maybe you add some of them and remove some other ....

Reply:
See also `CI / set_status (pull_request) Skipped `

Reply:
I have commented out a few of the lines and it seems to run now. Maybe we should thin out the Boost builds a bit in general in a separate change?

Reply:
That sounds like a plan